logagg/collect/collector.py

```diff
@@ -99,6 +99,25 @@ def validate_log_format(self, log):
         assert isinstance(log['handler'], str)
         assert isinstance(log['raw'], str)
 
+    def check_depth_at_nsq(self):
+        url = STATS_URL % (self.nsqd_http_address, self.nsqtopic)
+        data = self.session.get(url)
+        data = json.loads(data.content)
+        if 'health' in data:
+            topics = data.get('topics')
+            for record in topics:
+                topic_name = record['topic_name']
+                if self.nsqtopic in topic_name:
+                    get_depth_count = record.get('depth')
+                    return get_depth_count
+        else:
+            topics = data.get('data').get('topics')
+            for record in topics:
+                topic_name = record['topic_name']
+                if self.nsqtopic in topic_name:
+                    get_depth_count = record.get('depth')
+                    return get_depth_count
```

Comment:
@praneeth36 Are these 6 lines present in `else` condition is a duplicate piece of code as we have in `if` condition?